interface/orders/orders_results.php

```diff
@@ -281,7 +281,7 @@ function extShow(lino, show) {
 function prDateRequired(rlino) {
  var f = document.forms[0];
  if (f['form_date_report['+rlino+']'].value.length < 10) {
-  alert('<?php xl('Missing report date', 'e') ?>');
+  alert('<?php xlt('Missing report date', 'e') ?>');
```

Comment:
This is within javascript context, and we have a special function for translating within javascript context. It is the xlj() function, which basically is the same as js_escape(xl()). Note that the javascript escape functions (js_escape() and xlj()) will place quotes around the variable, so changing this involves using xlj and removing the quotes and will look like so (also not I removed the `e` parameter and added an `echo`):
```
alert(<?php echo xlj('Missing report date'); ?>);
```

Reply:
Got it, both the echo parameter and xlj fixed! :)